kubermatic/resource_node_deployment.go

```diff
@@ -35,7 +35,7 @@ func resourceNodeDeployment() *schema.Resource {
 			},
 			"name": {
 				Type:        schema.TypeString,
-				Computed:    true,
+				Required:    true,
```

Comment:
the name for node can be server side generated, please change to
```
Computed: true
Optional: true
```

Reply:
Creating node deployment without name gives 500 if cluster previously was created without node deployment. This is kubermatic error. I will add a TODO so make the field computed when bug is fixed. For now lets have it required.